### DISCUSSION TITLE
The left menu bar  empty，WHY?

### DISCUSSION THREAD

**Discussion by xuxer** [BIPES/BIPES#199] [General]
![image](https://user-images.githubusercontent.com/7438591/236890645-be6f06be-20b7-420a-82fe-13a3398e0c0f.png)
I downloaded the source code to my D drive, but when I opened the 'ui/index-offline.html', why is there no menu on the left? Same as the above picture

**Comment by jetannenbaum** [comment]
A few questions:
1. It looks like you are on a Windows computer, correct?
2. Which branch did you start from?
3. Which device did you select?
4. Did you successfully build the index_offline.html file? If so, what tool(s) did you use to build the file?
5. Can you see any of the devices in the index_offline.html starting around line 750 or so?  You should see a line like this in the file:
    <document style='display: none' id='OFFLINE_toolbox_arduino_xml'>

**Comment by jessyjones** [comment]
For what it's worth, I've encountered the same error, from what I understood in the logs, opening the index.html file (double clicking it) prevented the XML files from being loaded because of CORS policy. 
What solved this for me was to launch a simple webserver and load the page that way instead

**Comment by liuandy1207** [comment]
I made a change to the Makefile and I think that fixed this missing toolbox issue in index_offline.html.

Here's a link to my fork with that change: https://github.com/liuandy1207/BIPES-offline
